src/App.vue

```diff
@@ -9,7 +9,7 @@
             >
                 <i class="form-icon icon icon-menu mr-2" style="font-size: 1.3rem"></i>
             </a>
-            <img src="./assets/vechain-app-hub.png" style="height: 40px;" class="mr-2" alt="Logo" />
+            <img src="./assets/vechain-app-hub.png" class="mr-2" alt="Logo" id="logo-navbar" />
```

Comment:
please use tailwind classes instead of id

Reply:
I would still use classes instead of ids to style the code
https://stackoverflow.com/questions/3434278/do-dom-tree-elements-with-ids-become-global-properties